### ISSUE TITLE
Regression: MultiReporter fails when passing fail-on configuration to colorized reporter (PR #10444)

### ISSUE THREAD

**Issue by wenwen-666** [PyCQA/pylint#10905]
### Bug description

```python
PR #10444 added a feature to colorize --fail-on messages, but introduced a bug when using MultiReporter (e.g., output to both file and console).

Root cause (lines 539-541 in pylint/lint/pylinter.py):
elif isinstance(self.reporter, reporters.MultiReporter):
    for _reporter in self.reporter._sub_reporters:
        if isinstance(self.reporter, ColorizedTextReporter):  # ❌ Bug: checking wrong variable
            self.reporter.set_fail_on_symbols(self.fail_on_symbols)  # ❌ Bug: calling on wrong object
Should be:
elif isinstance(self.reporter, reporters.MultiReporter):
    for _reporter in self.reporter._sub_reporters:
        if isinstance(_reporter, ColorizedTextReporter):
            _reporter.set_fail_on_symbols(self.fail_on_symbols)

Regression test that reproduces the issue:
from pylint.lint.pylinter import PyLinter
from pylint.reporters import MultiReporter
from pylint.reporters.text import ColorizedTextReporter

linter = PyLinter()
linter.fail_on_symbols = {'missing-docstring'}
linter.set_reporter(MultiReporter([
    ColorizedTextReporter(),
    ColorizedTextReporter()  # Two color reporters to ensure loop logic works
]))
linter.pass_fail_on_config_to_color_reporter()  # Raises AttributeError
Before PR #10444: AttributeError: 'ColorizedTextReporter' object has no attribute 'set_fail_on_symbols' (expected - feature didn't exist)

After PR #10444: AttributeError: 'MultiReporter' object has no attribute 'set_fail_on_symbols' (regression - wrong variable used)

The PR's Codecov warnings flagged these lines as uncovered, confirming missing test coverage for MultiReporter scenarios.
```

### Configuration

```ini

```

### Command used

```shell
pylint --output-format=colorized --output=out.txt --fail-on=missing-docstring mymodule.py
```

### Pylint output

```python
AttributeError: 'MultiReporter' object has no attribute 'set_fail_on_symbols'
```

### Expected behavior

No error. The fail-on configuration should be passed to ColorizedTextReporter instances inside MultiReporter, and fail-on messages should be colorized red inverse as documented.

### Pylint version

```shell
Affected: All versions after PR #10444 (merged July 13, 2025)
pylint 3.3.0
astroid 3.3.0
Python 3.12.0
```

### Additional dependencies

```python

```

**Comment by DanielNoord** [comment]
I shouldn't have trusted you. This was already fixed https://github.com/pylint-dev/pylint/commit/217212fdeaeb911512ae25e6e611d8a71f31a375 as found by Copilot.

Going to block you, you have wasted enough of our time.
